FlightReservation/App.js

```diff
@@ -20,6 +25,11 @@ const App = () => {
         <Stack.Navigator screenOptions={{header: () => null}}>
           <Stack.Screen name="SignUp" component={SignUp} />
           <Stack.Screen name="LogIn" component={LogIn} />
+          <Stack.Screen name="From" component={From} />
+          <Stack.Screen name="To" component={To} />
+          <Stack.Screen name="Date" component={Date} />
+          <Stack.Screen name="Passenger" component={Passenger} />
+          <Stack.Screen name="Results" component={Results} />
```

Comment:
as a recommendation make an object array and then a map over that array

Reply:
Done